adoc/Release-Notes.adoc

```diff
@@ -31,6 +31,112 @@ endif::[]
+** Added feature flags to enable roles such as autoscaler, cf-usb, credhub and eirini.
+** Added Sync Integration Test Suite (SITS).
+** Added support for NGINX Ingress Controller with customizable Ingress via user suppplied annotations.
+** Added .net-core buildpack (2.2.7).
+* The Stratos UI has been updated to version 2.4:
+** For a full list of features and fixes see https://github.com/SUSE/stratos/releases/tag/2.4.0.
+
+For information about deploying and administering {product}, see the product manuals at
+{doc-url}.
+
+
+[id='sec.1_4.feature']
+==== Features and Fixes
+
+* cf-mysql-release has been pinned at version 36.15.0 to avoid intermittent database connectivity errors in HA setup.
+
+* Changed app autoscaler-postgres to a non-HA setup due to a known limitation - see https://github.com/cloudfoundry/postgres-release/#known-limitations.
+
+* The app autoscaler services are no longer deployed as {k8s} services of type LoadBalancer and therefore, are not exposed on public IP addresses or hostnames.
+
+* Fixed autoscaler to perform SSL validation.
+
+* Fixed autoscaler to listen to cluster internal CF API endpoint.
+
+* The default `nproc` limits for the vcap user for all SCF roles have been bumped to 1024/2048 (soft/hard). You can use different limits by setting `kube.limits.nproc.soft` and `kube.limits.nproc.hard` in the Helm chart values. 
+
+* Cleaned up role readiness probe outputs.
+
+* Fixed the test for an insecure Docker registry (uses tcpdomain for the route).
+
+* Includes these {cf} component versions:
+** app-autoscaler: 1.0.0
+** bits-service: 2.26.0
+** bpm: 1.0.0
+** capi: 1.79.0
+** cf-deployment: 6.10
+** cf-mysql: 36.15.0
+** cf-routing: 0.184.0
+** cf-sle12: 1.75.11
+** cf-smoke-tests: 40.0.44
+** cf-syslog-drain: 8.1
+** cf-usb: 1.0.1
+** cflinuxfs2: 1.281.0
+** cflinuxfs3: 0.81.0
+** credhub: 2.1.2
+** diego: 2.25.0
+** eirini: 0.0.4
+** garden-runc: 1.17.2
+** groot-btrfs: 1.0.4
+** kubectl: 1.9.6
+** loggregator: 104.4
+** loggregator-agent: 3.2
+** nats: 26
+** nfs-volume: 1.7.6
+** postgres-release: 26
+** scf-helper: 1.0.2
+** cf-acceptance-tests: 
+** statsd-injector: 1.5.0
+** uaa: 68.0
+* Buildpacks:
+** binary-buildpack: 1.0.32
+** dotnet-core-buildpack: 2.2.10
+** go-buildpack: 1.8.36
+** java-buildpack: 4.19.1 
+** nginx-buildpack: 1.0.11
+** nodejs-buildpack: 1.6.49
+** php-buildpack: 4.3.75
+** python-buildpack: 1.6.32
+** ruby-buildpack: 1.7.38
+** staticfile-buildpack: 1.4.42
+
+[id='sec.1_4.issue']
+==== Known Issues
+
+* The instructions for enabling Eirini can be found at https://github.com/SUSE/scf/wiki/Eirini.
+
+* Resuming a past practice, with {product} 1.4, use the complete command: `helm upgrade --force --recreate-pods` for an upgrade. This will reintroduce downtime for apps but without `--recreate-pods`, multiple versions of statefulsets may co-exist which can cause incompatibilities between depedent statefulsets, and result in a broken upgrade. This applies to Stratos pods as well.
```

Comment:
Offer instructions on mitigating downtime by pre-pulling the CAP 1.4 base images for scf and uaa to each Kubernetes node before running `helm upgrade ...`

Reply:
Need to do still in a new PR. According to Troy: 
`docker pull registry.suse.com/cap/scf-api-group:c4e92661127f706c3afbceeac23bd8e78349d4ab` and `docker pull registry.suse.com/cap/uaa-uaa:378dee28c60e7a7ac892d79ad8fa043d63c2c7cf`